src/sentry/static/sentry/app/components/stream/processingIssueList.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {isEqual} from 'lodash';
+import PropTypes from 'prop-types';
+
+import {Client} from 'app/api';
+import {logAjaxError} from 'app/utils/logging';
+import {fetchProcessingIssues} from 'app/actionCreators/processingIssues';
+import ProcessingIssueHint from 'app/components/stream/processingIssueHint';
+import SentryTypes from 'app/sentryTypes';
+
+class ProcessingIssueList extends React.Component {
+  static propTypes = {
+    organization: SentryTypes.Organization.isRequired,
+    projectIds: PropTypes.array,
+    showProject: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    showProject: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.api = new Client();
+    this.state = {
+      loading: true,
+      issues: null,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchIssues();
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (!isEqual(prevProps.projectIds, this.props.projectIds)) {
+      this.fetchIssues();
+    }
+  }
+
+  componentWillUnmount() {
+    this.api.clear();
+  }
+
+  fetchIssues() {
+    let {organization, projectIds} = this.props;
+
+    fetchProcessingIssues(this.api, organization.slug, projectIds).then(
+      data => {
+        let haveIssues = data.filter(
```

Comment:
I realise this is a copy and paste but I'm not a fan of this variable name - `haveIssues` implies a boolean to me